sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store/feature_store.py

```diff
@@ -0,0 +1,166 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=too-many-instance-attributes
+
+from typing import Dict, Optional
+
+from marshmallow import ValidationError
+
+from azure.ai.ml._restclient.v2022_12_01_preview.models import Workspace as RestWorkspace
+from azure.ai.ml.entities import Workspace, CustomerManagedKey, FeatureStoreSettings, ComputeRuntime
+from azure.ai.ml.entities._credentials import IdentityConfiguration, ManagedIdentityConfiguration
+from azure.ai.ml._utils._experimental import experimental
+
+from .materialization_store import MaterializationStore
+from ._constants import OFFLINE_STORE_CONNECTION_NAME, DEFAULT_SPARK_RUNTIME_VERSION, FEATURE_STORE_KIND
+
+
+@experimental
+class FeatureStore(Workspace):
```

Comment:
"Featurestore" should be one word, similar to "Datastore"

cc: @qjxu

Reply:
in cli it is like feature-store, so the word is like this

Reply:
@qjxu should we change it in cli as "featurestore"

Reply:
With sdk review, we agreed to use "FeatureStore" in sdk and "feature-store" in cli.